tutorials/how_to_dataloader.ipynb

```diff
@@ -0,0 +1,420 @@
+    "            \"M_sigma33_ppm\",\n",
+    "            \"E_sigma11_ppm\",\n",
+    "            \"E_sigma22_ppm\",\n",
+    "            \"E_sigma33_ppm\",\n",
+    "        ]\n",
+    "\n",
+    "data_loader = DataLoader(\n",
+    "    feature_columns=feature_columns,\n",
+    "    target_columns=\"metal_X4_E\",\n",
+    "    test_size=0.2,\n",
+    "    random_state=42,\n",
+    "    dataset_size=0.001,\n",
+    "    target_type=\"categorical\",  # can be \"categorical\" or \"one-hot\"\n",
+    "    complex_geometry=\"all\", # can be oct, spy, tbp or all\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load data\n",
+    "To load the dataset you can just call the `load_data()` methodhod. This will return the X and y tests and trains respectively and also the y_labels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_train, X_test, y_train, y_test, y_labels = data_loader.load_data()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[-0.2722084  -2.21144124 -1.43206187  3.74585072 -5.16278678 -3.93518974\n",
+      "   0.          0.          1.          0.        ]\n",
+      " [-0.18789275 -0.07571338 -0.56527957  0.58809239  0.38235406  0.57606602\n",
+      "   0.          1.          0.          3.        ]\n",
+      " [ 0.65276891  1.32677885  1.6089516   0.18998478  0.48501124  0.4769992\n",
+      "   0.          1.          4.          3.        ]\n",
+      " [-0.41815093  0.07191111  0.32900997 -0.80085373 -0.79536918 -0.56904129\n",
+      "   0.          1.          4.          2.        ]\n",
+      " [ 0.73145027  0.54911177  0.49709759  0.6091456   0.26643317  0.51446819\n",
+      "   0.          0.          1.          0.        ]\n",
+      " [ 0.99191313  1.31089452  0.81234606  0.10204876  0.48856963  0.56350613\n",
+      "   0.          9.          1.          2.        ]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(X_test)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[ 0 10 13]\n",
+      " [ 1  1  9]\n",
+      " [ 1  6 11]\n",
+      " [ 0  3 12]\n",
+      " [ 1  4  8]\n",
+      " [ 1  6  7]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(y_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Decode target (y) arrays\n",
+    "To decode the targets you got you can use ether the `categorical_target_decoder()` or the `binarized_target_decoder()` method of the dataloader. This will return a labled list of lists. As far as I know these decoded things can be easily passed into the cm or maybe after flattening them."
```

Comment:
What is a labeled list of lists? Not entirely sure what that means 😅 Would a mapping in a dictionary be better ?

Reply:
It's a list of lists (basically an array) that contains the Label in each position instead of a meaningless number. So "Mo" or "Ethyl" instead of 1 or 7

Reply:
Super, thanks! Maybe good to mention this in the docs with explicit examples